src/tools/clippy/tests/ui/allow_attributes.stderr

```diff
@@ -19,13 +19,5 @@ error: #[allow] attribute found
 LL |     #[allow(unused)]
    |       ^^^^^ help: replace it with: `expect`
 
-error: #[allow] attribute found
-  --> tests/ui/allow_attributes.rs:52:7
-   |
-LL |     #[allow(unused)]
-   |       ^^^^^ help: replace it with: `expect`
-   |
-   = note: duplicate diagnostic emitted due to `-Z deduplicate-diagnostics=no`
-
-error: aborting due to 4 previous errors
```

Comment:
Something is wrong here. This lint trigger for this has been removed:

```rs
#[clippy::msrv = "1.81"]
fn msrv_1_81() {
    #[allow(unused)]
    let x = 1;
}
```

Same for `allow_attributes_without_reason.rs`. I'm guessing that it has to do with the `clippy::msrv` attribute but I don't understand why. Both this and the old one use `extract_msrv_attr` just with a different context. They should be equivalent, and the MSRV check also looks good... Do you have any idea where this comes from?

Reply:
This seems to actually be a bug in the old behaviour – it emitted the same lint twice. I don't know how I fixed it but it seems to be better than before